tests/build/build.go

```diff
@@ -48,7 +48,7 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 	f, err := framework.NewFramework()
 	Expect(err).NotTo(HaveOccurred())
 
-	Describe("the component with git source (GitHub) is created", Ordered, Label("github-webhook"), func() {
+	Describe("the component with git source (GitHub) is created", Ordered, Label("github-webhook", "pipeline"), func() {
```

Comment:
This test is mostly about verifying the PaC integration.

If we're about to test the `component CR -> pipelinerun` flow, would it make more sense to use [this test](https://github.com/redhat-appstudio/e2e-tests/blob/fb11bd6ed59f37c08f54dead4856aacc5ccd45fb/tests/build/build.go#L448-L466) for pipeline-service testing instead? (i.e. that means to add the `pipeline` label where `buildTemplatesTestLabel` is)

Reply:
@psturc thank you for the reminder. We also need to verify the Pac integration, that's why I marked all Build tests as part of pipeline-service tests.